src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -106,6 +106,33 @@ <h2 class="h2 title" id="notifications">{% trans "Voorkeuren voor meldingen" %}
     <section class="profile-section profile-section__overview">
         <h2 class="h2 title" id="profile-overview">{% trans "Overzicht" %}</h2>
         <div class="profile-cards card-container card-container--columns-3">
+            {# Selected categories #}
+            {% if view.config.selected_categories %}
+                <a href="{% url 'profile:categories' %}" class="card card--compact card--stretch" id="profile-selected-categories">
+                    <div class="card__body">
+                        <div class="profile__link">
+                            <p class="card__heading-4"><span class="link link__text">{% trans "Mijn Interessegebieden" %}</span></p>
```

Comment:
Please change the `<p>` tag for the card heading into an actual `<H4>` heading tag for accessibility  - that's the last thing before I'll approve :-)
(PS: I _actually_ see the cardheadings on this page all need to be H3 tags with that .card__heading-4 class... but that may be part of a future accessibility sprint)